roles/nginxsites/tasks/main.yml

```diff
@@ -61,10 +193,46 @@
   when: >
     (("all" in domains_to_use and "none" in domains_to_skip) or item.0.name in domains_to_use)
     and item.0.name not in domains_to_skip
+    and (item.0.wp.install is undefined or item.0.wp.install == 'singlesite')
   with_nested:
     - "{{ domains }}"
     - "{{ nginx_defaults.single_wp_site }}"
 
+## Modify access_log_path directive
+- name: Modify access_log path directive for configurations. (for a sub-directory multisite install)
+  lineinfile:
+    dest: "{{ nginx_defaults.file_path}}/{{ item.0.name }}.{{ item.1.type }}.conf"
+    regexp: "access_log"
+    line: "access_log /home/{{ remote_web_user }}/www/{{ item.0.name }}/logs/access.log;"
+    state: present
+  when: >
+    (("all" in domains_to_use and "none" in domains_to_skip) or item.0.name in domains_to_use)
+    and item.0.name not in domains_to_skip
+    and item.0.wp.install is defined
+    and item.0.wp.install == 'multisite'
+    and (item.0.wp.network_type is undefined or item.0.wp.network_type == 'subdirectory')
+  with_nested:
+    - "{{ domains }}"
+    - "{{ nginx_defaults.subdirectory_multi_wp_site }}"
+
+## Modify access_log_path directive
+- name: Modify access_log path directive for configurations. (for a sub-domain multisite install)
+  lineinfile:
+    dest: "{{ nginx_defaults.file_path}}/{{ item.0.name }}.{{ item.1.type }}.conf"
+    regexp: "access_log"
+    line: "access_log /home/{{ remote_web_user }}/www/{{ item.0.name }}/logs/access.log;"
+    state: present
+  when: >
+    (("all" in domains_to_use and "none" in domains_to_skip) or item.0.name in domains_to_use)
+    and item.0.name not in domains_to_skip
+    and item.0.wp.install is defined
+    and item.0.wp.install == 'multisite'
+    and item.0.wp.network_type is defined
+    and item.0.wp.network_type == 'subdomain'
+  with_nested:
+    - "{{ domains }}"
+    - "{{ nginx_defaults.subdomain_multi_wp_site }}"
```

Comment:
hmm, I'm wondering if types (ssl/nonssl) should be a property of the `domains` object.   The `vars.yml` should probably be what defines the the type of site installed (type meaning ssl and nonssl).  We shouldn't assume that everytime a site is setup its going to be set up for both ssl and nonssl right?  Although, I guess this is just setting up the `sites-available` configs so its likely okay.  

You can leave as is, just a thought I had.

Reply:
Right, right now it seems that both types of nginx configs are created for each site, but I didn't touch that as thought that that's intended.